packages/@tinacms/fields/src/Wysiwyg/node-views/ImageView/image.css

```diff
@@ -0,0 +1,22 @@
+/**
+
+Copyright 2019 Forestry.io Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+.tinacms-image {
+  display: inline-block;
+  padding: 1em;
+}
```

Comment:
Padding will be applied to image and not image wrapper.

Reply:
What if we had `margin: 1em 0;` and 0 padding? (outline right on the image, no horizontal spacing between image and margins)

Not sure I like that the image isn't left aligned to the text; is that only when the focus state is applied? We don't necessarily have to address all the design concerns in this PR, I can tweak the styles later as well.

Reply:
Hey @spbyrne , I have done this change to styles. You can do any more required style changes to images here: https://github.com/tinacms/tinacms/pull/570/files#diff-85a5bec6b762da44b21a435e5fc7c8bdR21

Reply:
Sweet, thanks! 👍